spec/unit/jobs/services/service_instance_state_fetch_spec.rb

```diff
@@ -387,6 +387,66 @@ def run_job(job)
                 end
               end
             end
+            context 'when exponential backoff is not set to default' do
+              before do
+                TestConfig.config[:broker_client_async_poll_exponential_backoff_rate] = 2.0
+              end
+
+              describe 'it updates the polling interval' do
+                it 'when changing exponential backoff rate only' do
+                  enqueued_time = 0
+
+                  Timecop.freeze do
+                    run_job(job)
+                    enqueued_time = Time.now
+                  end
+
+                  [60, 180, 420, 900, 1860].each do |seconds|
+                    Timecop.freeze((seconds - 1).seconds.after(enqueued_time)) do
+                      execute_all_jobs(expected_successes: 0, expected_failures: 0)
+                    end
+
+                    Timecop.freeze((seconds + 1).seconds.after(enqueued_time)) do
+                      execute_all_jobs(expected_successes: 1, expected_failures: 0)
+                    end
+                  end
+                end
+
+                it 'when changing exponential backoff rate and default polling interval' do
+                  TestConfig.config[:broker_client_default_async_poll_interval_seconds] = 10
+
+                  enqueued_time = 0
+
+                  Timecop.freeze do
+                    run_job(job)
+                    enqueued_time = Time.now
+                  end
+
+                  [10, 30, 70, 150, 310].each do |seconds|
+                    Timecop.freeze((seconds - 1).seconds.after(enqueued_time)) do
+                      execute_all_jobs(expected_successes: 0, expected_failures: 0)
+                    end
+
+                    Timecop.freeze((seconds.ceil + 1).seconds.after(enqueued_time)) do
+                      execute_all_jobs(expected_successes: 1, expected_failures: 0)
+                    end
+                  end
+                end
+              end
+
+              it 'it takes the exponential backoff into account when checking whether the next run would exceed the maximum duration' do
```

Comment:
The `it` string can be omitted, i.e. `it 'takes ...'` instead of `it 'it takes ...'`